docs/topics/gradle/gradle-configure-project.md

```diff
@@ -37,12 +37,13 @@ plugins {
 </tab>
 </tabs>
 
-When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions:
+When configuring your project, check the Kotlin Gradle plugin compatibility with available Gradle versions. 
+In the following table, there are minimum and maximum fully supported versions of Gradle and Android Gradle plugin:
 
-|                       | Minimum supported version | Maximum fully supported version |
-|-----------------------|---------------------------|---------------------------------|
-| Gradle                | %minGradleVersion%        | %maxGradleVersion%              |   
-| Android Gradle plugin | %minAndroidGradleVersion% | %maxAndroidGradleVersion%       |
+| Kotlin version | Gradle min and max versions              | Android Gradle plugin min and max versions            |
```

Comment:
I am not sure about "max versions" - sounds like greater version will not work :thinking: 

Could we for now use "max fully supported versions"?

Reply:
This is written in the sentence just before the table:
>In the following table, there are minimum and maximum fully supported versions of Gradle and Android Gradle plugin:

I can mark fully supported versions with bold. Does it work?

Of course, I can add words "fully supported" to the table itself, but I avoided it for not making the table header too large.

Reply:
> I can mark fully supported versions with bold. Does it work?

Nah, people will ignore it. Better then to add also following sentence:
> Latest Gradle and AGP versions should generally work without issues.

Reply:
Ok, I'll add. Now it looks so (tried bold). This is a preview from IDEA, on stagings %kotlinVersion% will be replaced with 1.8.0, but builds don't work right now😁
<img width="866" alt="Screenshot 2022-12-21 at 22 00 48" src="https://user-images.githubusercontent.com/78360457/209001975-37a5bf9a-6d10-4caa-9b19-e43b64ff3dfb.png">